src/screens/Customers/components/CustomerCard.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { Card, CardActions, Typography, IconButton } from '@material-ui/core';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    arrow: {
```

Comment:
I actually don't see a big difference if i delete this class for `ArrowForwardIosIcon` and instead just put `size='small'`. I think it's the correct color by default, but lmk if I'm wrong.